src/Backend/Modules/ContentBlocks/ContentBlock/ContentBlockRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\ContentBlock;
+
+use Backend\Core\Language\LanguageName;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Query;
+
+class ContentBlockRepository extends EntityRepository
+{
+    /**
+     * @param LanguageName $language
+     *
+     * @return Query
+     */
+    public function getDataGridQuery(LanguageName $language)
+    {
+        return $this->getEntityManager()
+            ->createQueryBuilder()
+            ->select('cb.id, cb.title, cb.isHidden')
+            ->from(ContentBlock::class, 'cb')
+            ->where('cb.status = ?1 AND cb.language = ?2')
```

Comment:
I prefer named parameters. I think it's also cleaner to use two where statements when using doctrine queryBuilders:

```php
->where('cb.status = :status')
->andWhere('cb.language = :language')
```